mllib/src/test/scala/org/apache/spark/mllib/tree/GradientBoostedTreesSuite.scala

```diff
@@ -111,8 +115,10 @@ class GradientBoostedTreesSuite extends SparkFunSuite with MLlibTestSparkContext
           EnsembleTestHelper.validateClassifier(gbt, GradientBoostedTreesSuite.data, 0.9)
         } catch {
           case e: java.lang.AssertionError =>
+            // scalastyle:off println
             println(s"FAILED for numIterations=$numIterations, learningRate=$learningRate," +
```

Comment:
Hm, I wonder if that is what Marcelo juuust fixed in https://github.com/apache/spark/pull/7140